tests/tantivy_test.py

```diff
@@ -925,3 +925,15 @@ def test_document_snippet(self, dir_index):
             assert first.end == 23
             html_snippet = snippet.to_html()
             assert html_snippet == "The Old Man and the <b>Sea</b>"
+
+
+class TestQuery(object):
+    def test_term_query(self, ram_index):
+        index = ram_index
+        query = Query.new_term_query(index.schema, "title", "sea")
```

Comment:
I think exposing the fundamental query types is not too useful without having combinators, i.e. we should at least add `union` and `intersection` combinators to combine queries on the Python side.

Reply:
I guess you mean BooleanQuery by "union" and "intersection"? Exposing TermQuery itself may not help for real use cases. However, I'd like to take baby steps and not bloat this PR. 
We can support all other queries in a step-by-step manner once this is successfully merged.

Reply:
Yes, they would be implemented via `BooleanQuery`, but mirror its API, there would most likely be `union` and `intersection` methods on `Query`.

Fair enough on doing this in a follow-up PR.

Reply:
Sure, we should be able to have some fluent APIs to construct a bool query. It'll be a future work; anyway, thanks for your suggestion!